internal/sql/ast/range_subselect.go

```diff
@@ -2,6 +2,7 @@ package ast
 
 type RangeSubselect struct {
```

Comment:
This type is based on the type here (https://pkg.go.dev/github.com/pganalyze/pg_query_go/v2#RangeSubselect), which does not include join information.